5calls/app/src/main/res/layout/activity_issue.xml

```diff
@@ -2,192 +2,96 @@
-                android:textColor="@color/colorAccent"
-                android:textSize="@dimen/h2_text_size"
-                android:visibility="gone"
-                />
+                style="@style/issueNameTextStyle" />
 
             <TextView
                 android:id="@+id/issue_description"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/activity_horizontal_margin"
-                android:paddingLeft="@dimen/activity_horizontal_margin"
-                android:paddingRight="@dimen/activity_horizontal_margin"
-                android:textColor="@color/colorPrimaryDark"
-                android:textSize="16sp"
-                />
+                style="@style/issueDescriptionTextStyle" />
 
         </LinearLayout>
 
     </android.support.v4.widget.NestedScrollView>
 
     <android.support.v4.widget.NestedScrollView
         android:id="@+id/bottom_sheet"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="@color/colorAccent"
-        android:elevation="16dp"
+        style="@style/nestedScrollViewStyleDown"
         android.support.design:ignore="UnusedAttribute"
         app:behavior_hideable="false"
         app:behavior_peekHeight="52dp"
-        app:layout_behavior="@string/bottom_sheet_behavior"
-        >
+        app:layout_behavior="@string/bottom_sheet_behavior">
 
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical"
-            >
+        <LinearLayout style="@style/issueLinearLayout">
 
             <RelativeLayout
                 android:id="@+id/rep_prompt"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
+                style="@style/issueRelativeLayoutStyle"
                 android:background="@color/colorAccent"
-                android:minHeight="52dp"
-                android:paddingLeft="@dimen/activity_horizontal_margin"
-                android:paddingRight="@dimen/activity_horizontal_margin"
-                >
+                android:minHeight="@dimen/issue_height_relative_layout">
 
                 <ImageView
                     android:id="@+id/expand_contacts_icon"
-                    android:layout_width="36dp"
-                    android:layout_height="36dp"
-                    android:layout_alignParentEnd="true"
-                    android:layout_alignParentRight="true"
-                    android:layout_centerVertical="true"
-                    android:contentDescription="@null"
-                    android:rotation="180"
-                    android:scaleType="center"
-                    android:src="@drawable/ic_expand_more_white_24dp"
-                    />
+                    style="@style/issueImageView" />
```

Comment:
Please keep the contentDescription here. It's not really a style thing and I think removing it can make android lint mad.